docs/reference/settings/notification-settings.asciidoc

```diff
@@ -245,31 +281,37 @@ via Slack. You can specify the following Slack account attributes:
 --
 [[slack-account-attributes]]
 
-  `secure_url` (<<secure-settings,Secure>>);;
-  The Incoming Webhook URL to use to post messages to Slack. Required.
-
-  `message_defaults.from`;;
-  The sender name to display in the
-                                    Slack message. Defaults to the watch ID.
-
-  `message_defaults.to`;;
-  The default Slack channels or groups you
-                                             want to send messages to.
-
-  `message_defaults.icon`;;
-  The icon to display in the Slack messages.
-                                             Overrides the incoming webhook's configured
-                                             icon. Accepts a public URL to an image.
-
-  `message_defaults.text`;;
-  The default message content.
-
-  `message_defaults.attachment`;;
-  Default message attachments. Slack message attachments
-                                    enable you to create more richly-formatted messages.
-                                    Specified as an array as defined in the
-                                    https://api.slack.com/docs/attachments[
-                                   Slack attachments documentation].
+`secure_url`::
+(<<secure-settings,Secure>>)
+The Incoming Webhook URL to use to post messages to Slack. Required.
+
+`message_defaults.from`::
+(<<static-cluster-setting,Static>>)
```

Comment:
`xpack.notification.slack.account.***.message_defaults` is listed as dynamic in the code. Is this from setting overridden to be static only?